doc/thing/samples/sensor-samples/dht11.rst

```diff
@@ -0,0 +1,24 @@
+dht
```

Comment:
This file is duplicated. Please, remove it.

Reply:
Done